dbt/models/shared_columns.md

```diff
@@ -210,6 +210,14 @@ This is the value after the first round of appeals at the Assessor's
 Office **and** the second round of appeals at the Board of Review.
 {% enddocs %}
 
+## calc_meth
+
+{% docs shared_column_calc_meth %}
+Calculation parameter.
+
+If present, must be `E`.
+{% enddocs %}
```

Comment:
These docs are not the clearest, but they correspond to what Mirella sent me over email.